common-multipart-utils-os/.gitignore

```diff
@@ -0,0 +1,38 @@
+# Compiled class file
+*.class
+
+# Log file
+*.log
+
+# BlueJ files
+*.ctxt
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.nar
+*.ear
+*.zip
+*.tar.gz
+*.rar
+
+# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
+hs_err_pid*
+/target/
+**/target/
+
+# IntelliJ
+.idea/
+*.iml
+
+# Eclipse
+**/.classpath
+**/.project
+**/org.eclipse.*
+.factorypath
```

Comment:
I decided to go with the .gitignore convention that the original projects had because I wanted to keep things as consistent as possible so that it will make it easier for the common services team if/when they take over these projects as things will match their conventions.